### PR TITLE
Make rails responsive on public event page

### LINKED ISSUE #30: Add a small button on the top which toggles left side rail on event description page.
The left side rail on the event description page hides from the screen and is not visible in case of mobile screens( < 768px ).

Current behaviour : 
mobile

Expected behaviour : 
mobile4

Above picture shows the sample thing we want to do. Add a small button at the page wherever suitable which acts as a dropdown to the menu on the event description page.

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Replaces the rails with standard menu to make it stackable and responsive on the mobile
#### Changes proposed in this pull request:

-
-
-
giphy

Fixes #30

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
@niranjan94 Please Review

File: app/templates/public.hbs [hunk 1]
```diff
@@ -15,18 +15,25 @@
...
           {{/ui-sticky}}
-        </div>
+        {{/if}}
       </div>
-      <div>
+      <div class="ten wide column">
         {{outlet}}
       </div>
-      <div class="right ui rail">
+      <div class="three wide column" style="height:100vh;overflow: hidden; !important">
```
Comment (CosmicCoder96) [inline app/templates/public.hbs]: the styling is a temporary-fix to prevent content overflow in the twitter widget.

**Comment by CosmicCoder96** [comment]
@niranjan94 Thank you. I have made the changes, please review again.

**Comment by niranjan94** [comment]
@CosmicCoder96 please fix the tests

**Comment by CosmicCoder96** [comment]
@niranjan94 done, have updated the test.

**Pull request merged**